lib/ansible/config/base.yml

```diff
@@ -271,15 +271,19 @@ COLLECTIONS_ON_ANSIBLE_VERSION_MISMATCH:
   ini: [{key: collections_on_ansible_version_mismatch, section: defaults}]
   choices: [error, warning, ignore]
   default: warning
+_COLOR_DEFAULTS: &color
+  name: placeholder for color settings' defaults
+  allowed: ['black', 'bright gray', 'blue', 'white', 'green', 'bright blue', 'cyan', 'bright green', 'red', 'bright cyan', 'purple', 'bright red', 'yellow', 'bright purple', 'dark gray', 'bright yellow', 'magenta', 'bright magenta', 'normal']
```

Comment:
Wouldn't using `choices` be the same thing?

Reply:
duh!